teos/src/api/http.rs

```diff
@@ -217,6 +218,18 @@ async fn get_subscription_info(
     Ok(reply::with_status(body, status))
 }
 
+async fn ping(
+    addr: Option<SocketAddr>,
+) -> Result<impl Reply, Rejection> {
+    log::debug!(
+        "Received a ping request from {}",
+        addr.map_or("an unknown address".to_owned(), |a| a.to_string())
+    );
+    Ok(Response::builder()
+        .status(StatusCode::OK)
+        .body::<hyper::Body>("".into()))
```

Comment:
Do we have to attach a body?

Reply:
I think this also brought the `hyper` dependency

Reply:
oh yes.. Actually i was in confusion should be we use or not. yes i have to use `hyper` dependency due to it!

Reply:
what should I do?? should I remove it or leave it as it is?

Reply:
Yeah, definitely do not add a dependency for something like this.

`warp` has a method to create HTTP 200 OK responses with empty bodies.

Reply:
Okayy Done! Thanks